docs/release-notes.7.0.md

```diff
@@ -1,5 +1,12 @@
 # Qooxdoo Release Notes
 
+## Fixes and optimizations for v7.7.3
+In this release we did some optimizations:
+   - add a cache for label size
+   - use Map instead of Array to register event listeners
+   - prevent .placeToWidget(..., true) from eating cpu
```

Comment:
- Feature: Allow runtime environment mutations for testing by setting environment settint "qx.environment.allowRuntimeMutations"=true  (implements #10747)
- Fix: qx.io.jsonrpc.Client throws uncatchable error on server-side error (#10739)